packages/ember-simple-auth-oauth2/lib/simple-auth-oauth2/authenticators/oauth2.js

```diff
@@ -218,13 +231,25 @@ export default Base.extend({
     @protected
   */
   makeRequest: function(url, data) {
-    return Ember.$.ajax({
-      url:         url,
-      type:        'POST',
-      data:        data,
-      dataType:    'json',
+    var _this = this;
+    var options = {
+      url: url,
+      type: 'POST',
+      data: data,
+      dataType: 'json',
       contentType: 'application/x-www-form-urlencoded'
```

Comment:
Can you line those values up again - actually the way it was formatted is kind of ESA's "official" style™